### ISSUE TITLE
arm64 redis-opertor run error

### ISSUE THREAD

**Issue by hongshenggit** [OT-CONTAINER-KIT/redis-operator#347]
**What version of redis operator are you using?**
latest
```shell
kubectl logs <_redis-operator_pod_name> -n <namespace>
```
![image](https://user-images.githubusercontent.com/87966434/194744772-fcfd9116-e8fa-45bf-988b-ad8e716fc3f7.png)

redis-operator version: latest

**Does this issue reproduce with the latest release?**
yes

**What operating system and processor architecture are you using (`kubectl version`)?**
1.23.7

<details><summary><code>kubectl version</code> Output</summary><br><pre>
$ kubectl version

</pre></details>

**What did you do?**
![image](https://user-images.githubusercontent.com/87966434/194744730-a5e2fdd3-bced-49af-ba3f-5e0b8f009198.png)

I build images on the arm64 machine (docker build -t <image_name> . )

helm install redis-operator -n <namespace> .
redis-operator works Running at first

When I do the following
helm install redis-cluster -n <namespace> .

redis-operator always restart
The log is as shown in the figure above

Question 1
Can the arm64 image be made in this way?

Question 2
Please tell me where the problem is

**Comment by hongshenggit** [comment]
The image I made using “make docker-build” also reported an error

**Comment by iamabhishek-dubey** [comment]
Can I see the YAML definition for this Redis cluster?

**Comment by hongshenggit** [comment]
![image](https://user-images.githubusercontent.com/87966434/194822721-b32cb4f3-04ca-4209-a099-7071ab836b94.png)
@iamabhishek-dubey There should be no problem

**Comment by devkmsg** [comment]
I don't think this is related to arm64, I can reproduce this on amd64 too.  This is always going to fail when `spec.persistenceEnabled` is unset as there is no default value.

https://github.com/OT-CONTAINER-KIT/redis-operator/blob/8de4f150976bc9b8e573e81abd94307f996da214/k8sutils/redis-cluster.go#L86

**Comment by iamabhishek-dubey** [comment]
> I don't think this is related to arm64, I can reproduce this on amd64 too. This is always going to fail when `spec.persistenceEnabled` is unset as there is no default value.
> 
> https://github.com/OT-CONTAINER-KIT/redis-operator/blob/8de4f150976bc9b8e573e81abd94307f996da214/k8sutils/redis-cluster.go#L86

This change is introduced in v0.12.0 but I guess @hongshenggit is using v0.11.0 because v0.12.0 is released today itself.

**Comment by iamabhishek-dubey** [comment]
I think when you compiled the binary, at that time there were some changes happening in the master branch. Can you please check for the latest examples for defining the manifest file and compile the binary again if needed?

**Comment by hongshenggit** [comment]
> > I don't think this is related to arm64, I can reproduce this on amd64 too. This is always going to fail when `spec.persistenceEnabled` is unset as there is no default value.
> > https://github.com/OT-CONTAINER-KIT/redis-operator/blob/8de4f150976bc9b8e573e81abd94307f996da214/k8sutils/redis-cluster.go#L86
> 
> This change is introduced in v0.12.0 but I guess @hongshenggit is using v0.11.0 because v0.12.0 is released today itself.

However, I have set the default SC
